questionnaires/phq8_gad7/phq8_gad7_armt.json

```diff
@@ -0,0 +1,572 @@
+[
+    {
+        "field_name": "phq8_gad7_1",
+        "form_name": "phq8_gad7",
+        "section_header": "Over the past two weeks, how often have you been bothered by any of the following problems ",
+        "field_type": "radio",
+        "field_label": "1. Little interest or pleasure in doing things?",
```

Comment:
probably don't need numbering here (unless presenting all Qs on the same screen)